Automattic/wp-calypso
# Marketing: Remove the Google AdWords card.

This PR removes the Google AdWords card featured on the Marketing and Integrations page at `/marketing/tools/:site`.

See pb5gDS-Ej-p2.

Before | After
--- | ---
![Jun-25-2020 16-05-22](https://user-images.githubusercontent.com/349751/86388336-7df40500-bc49-11ea-90d7-915c60588014.png) | ![Jun-29-2020 14-37-20](https://user-images.githubusercontent.com/349751/86388342-80eef580-bc49-11ea-99c8-3cae5fbc3c70.png)


#### Testing instructions

* Load this branch locally, or use the `calypso.live` link, and go to `/marketing/tools`.
* Verify the Google AdWords card is not displayed, for any type of site.
* Verify no other references to the card component remain in the Calypso codebase.
